openedx/features/edly/context_processor.py

```diff
@@ -27,20 +26,16 @@ def dynamic_theming_context(request):  # pylint: disable=unused-argument
     """
     Context processor responsible for dynamic theming.
     """
-    theming_context = cache.get(CACHE_NAME)
-    cache_timeout = configuration_helpers.get_value('EDLY_CACHE_TIMEOUT', DEFAULT_EDLY_CACHE_TIMEOUT)
```

Comment:
remove the defined variable `DEFAULT_EDLY_CACHE_TIMEOUT`

Reply:
`DEFAULT_EDLY_CACHE_TIMEOUT` is also used in `edly_app_context` function.

Reply:
I believe we should also remove caching from `edly_app_context` as well. Since we cannot afford have different sites branding and context for our site.
What do you suggest @asadmanzoor93 @rehan99000 ?

Reply:
Yes, we should remove the cache from the `edly_app_context`

Reply:
I will second this. We can't have different caching context against each client. The kindly guide is there any issues if we use the current same caching against all clients?

Reply:
I suppose if we have removed the cache from everywhere else, it doesn't make sense to leave one here.

Reply:
If we are going to remove cache altogether from all other locations then remove it from here as well.

Reply:
@asadmanzoor93 The cache is set on 2 places. One in dynamic theme context which we already removed. and the other is in `edly_app_context `

Reply:
@burhandodhy if there are no specific good user cases served by this caching then we can remove this as well.

Reply:
@asadmanzoor93 its removed.